openapi.yaml

```diff
@@ -301,6 +301,10 @@ definitions:
           type: integer
         inclusionProof:
           $ref: '#/definitions/InclusionProof'
+        signature:
+          type: string
+          format: byte
+          description: signature over the log entry (acts as an inclusion promise if inclusion proof is not included)
```

Comment:
right now Rekor is blocking on the insertion loop so this wouldn't really be a promise... but more of a static attestation of inclusion. Not sure how much to stress over the semantics here.

Reply:
Right, I was thinking we might want to leave it open for interpretation later in case we ever need to decouple these. I'm fine keeping it simple though.

Reply:
sgtm, I made the description more general here